tuned/plugins/plugin_usb.py

```diff
@@ -8,7 +8,23 @@
 
 class USBPlugin(base.Plugin):
 	"""
-	Plugin for tuning various options of USB subsystem.
+	`usb`::
+	
+	Sets autosuspend timeout of USB devices to the value specified by the
+	[option]`autosuspend` option in seconds. If the [option]`devices`
+	option is specified, the [option]`autosuspend` option applies to only
+	the USB devices specified, otherwise it applies to all USB devices.
+	+
+	The value `0` means that autosuspend is disabled.
+	+
+	.Turn USB autosuspend off for USB devices `1-1` and `1-2`
```

Comment:
To turn off USB autosuspend for USB devices `1-1` and `1-2`

Reply:
Interesting, the sentence sounded good to me as it was, but changed as you prefer.